app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -53,6 +53,13 @@ def find_by_installable_errata(_key, operator, value)
             { :conditions => "#{::Host::Managed.table_name}.id IN (#{facets.pluck(:host_id).join(',')})" }
           end
         end
+
+        def in_content_view_environment(content_view = nil, lifecycle_environment = nil)
```

Comment:
As a method signature it's not very easy to read (I have to go look up the method) to know what I might be nil-ing the value of, bit of a nitpick I suppose but would make this easier to read in places where one or the other parameter is nil.